mlrun/model_monitoring/db/stores/__init__.py

```diff
@@ -111,10 +111,12 @@ def get_store_object(
     ):
         store_type = mlrun.common.schemas.model_monitoring.ModelEndpointTarget.SQL
         kwargs["store_connection_string"] = store_connection_string
+    elif store_connection_string and store_connection_string == "v3io":
+        store_type = (
+            mlrun.common.schemas.model_monitoring.ModelEndpointTarget.V3IO_NOSQL
+        )
     else:
-        # Set the default store type if no connection has been set
-        store_type = mlrun.mlconf.model_endpoint_monitoring.store_type
-
+        store_type = None
```

Comment:
In that case the user will probably get an error when calling any of the model monitoring APIs. The issue is that it can breaks some APIs in the UI in which instead of returning "nothing", the user will get an error (e.g. model endpoints tab before setting mm creds).
We need to handle this issue by catching it on the BE side and return empty values without going into this flow.

Reply:
never mind I think you already handled it in https://github.com/mlrun/mlrun/pull/5827/files#:~:text=endpoints%20%3D%20schemas.ModelEndpointList(endpoints%3D%5B%5D) , just need to make sure it works without errors